custom/enikshay/nikshay_datamigration/factory.py

```diff
@@ -129,6 +128,15 @@ def get_person_case_structure(self):
             },
         }
 
+        if self._location:
+            assert self._location.location_type.name == 'phi'
+            kwargs['attrs']['owner_id'] = self._location.location_id
+        else:
+            kwargs['attrs']['owner_id'] = '_archive_'
+            kwargs['attrs']['update']['migration_archive_reason'] = 'migration_location_not_found'
```

Comment:
this should still be `archive_reason` since that is the property we are going to be using to figure out why the cases were archived